### PR TITLE
All Wiki links now use the consolidated Servarr wiki

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Title.

#### Todos
- [x] Tests 
-- [ ] Fix: src\NzbDrone.Core.Test\HealthCheck\HealthCheckFixture.cs
- [x] Update Remaining Links pending wiki Pages

### CODE REVIEW

File: frontend/src/AddSeries/AddNewSeries/AddNewSeries.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ class AddNewSeries extends Component {
                 <div className={styles.noResults}>Couldn't find any results for '{term}'</div>
                 <div>You can also search using TVDB ID of a show. eg. tvdb:71663</div>
                 <div>
-                  <Link to="https://github.com/Sonarr/Sonarr/wiki/FAQ#why-cant-i-add-a-new-series-when-i-know-the-tvdb-id">
+                  <Link to="https://wiki.servarr.com/Sonarr_FAQ#Why_can.E2.80.99t_I_add_a_new_series_when_I_know_the_TVDB_ID.3F">
                     Why can't I find my show?
```
Comment (markus101) [inline frontend/src/AddSeries/AddNewSeries/AddNewSeries.js:158]: These URLs look so random

File: frontend/src/Settings/Indexers/Options/IndexerOptions.js [hunk 2]
```diff
@@ -89,7 +89,7 @@ function IndexerOptions(props) {
                 unit="minutes"
                 helpText="Interval in minutes. Set to zero to disable (this will stop all automatic release grabbing)"
                 helpTextWarning="This will apply to all indexers, please follow the rules set forth by them"
-                helpLink="https://github.com/Sonarr/Sonarr/wiki/RSS-Sync"
+                helpLink="https://wiki.servarr.com/Sonarr_FAQ#How_does_Sonarr_find_episodes.3F"
```
Comment (markus101) [inline frontend/src/Settings/Indexers/Options/IndexerOptions.js:92]: Even the question marks are encoded instead of dropped 😢

**Review by markus101 — COMMENTED** [review]
Less about the changes and more about the crappy URLs

File: frontend/src/AddSeries/AddNewSeries/AddNewSeries.js (hunk 1 above)
Comment (bakerboy448) [reply frontend/src/AddSeries/AddNewSeries/AddNewSeries.js:158]: Gonna have to blame media wiki there; I suppose we can clean it up and encode on front end?

File: frontend/src/Settings/Indexers/Options/IndexerOptions.js (hunk 2 above)
Comment (bakerboy448) [reply frontend/src/Settings/Indexers/Options/IndexerOptions.js:92]: I know 😅

Comment (Qstick) [reply frontend/src/Settings/Indexers/Options/IndexerOptions.js:92]: Doesn't `https://wiki.servarr.com/Sonarr_FAQ#How_does_Sonarr_find_episodes` work just the same?

Comment (bakerboy448) [reply frontend/src/Settings/Indexers/Options/IndexerOptions.js:92]: Nope.  It doesn't bring you down to the section..."?" Get converted funky into 3F

**Comment by Drazzilb08** [comment]
All links labeled with `can_t` are now `cant` the underscore replacing the `'` has been removed

**Comment by bakerboy448** [comment]
> All links labeled with `can_t` are now `cant` the underscore replacing the `'` has been removed

done

**Pull request merged**